commons-geometry-euclidean/src/main/java/org/apache/commons/geometry/euclidean/threed/Vector3D.java

```diff
@@ -399,16 +399,24 @@ public boolean eq(final Vector3D vec, final Precision.DoubleEquivalence precisio
 
     /**
      * Get a hashCode for the vector.
-     * <p>All NaN values have the same hash code.</p>
-     *
+     * <p>
+     * All NaN values have the same hash code.
      * @return a hash code value for this object
      */
     @Override
     public int hashCode() {
         if (isNaN()) {
             return 642;
         }
-        return 643 * (164 * Double.hashCode(x) + 3 * Double.hashCode(y) + Double.hashCode(z));
+        int result = 17;
+        long temp;
+        temp = Double.doubleToLongBits(x);
```

Comment:
I thought this was fixed with a different linear combination of `Double.hashCode`. This seems to have reverted to using `doubleToLongBits`.